readthedocs/core/urls/__init__.py

```diff
@@ -1,20 +0,0 @@
-"""URL configuration for core app."""
-
-from __future__ import absolute_import
-from django.conf.urls import re_path
-
-from readthedocs.constants import pattern_opts
-from readthedocs.core import views
-
-
-core_urls = [
-    # Random other stuff
-    re_path(
-        (
-            r'^wipe/(?P<project_slug>{project_slug})/'
-            r'(?P<version_slug>{version_slug})/$'.format(**pattern_opts)
-        ),
-        views.wipe_version,
-        name='wipe_version',
-    ),
-]
```

Comment:
This is part of a decision that I've made to delete related non-used/invalid code together with these changes. The wipe "feature" is not a feature anymore because we do a full clean after the build is finished (`RTD_CLEAN_AFTER_BUILD=True`). So, wiping the environment has no effect and it was a confusing action for our users.

Reply:
Makes sense! I'm +1 on the removal in this PR. I've also removed this from the new site theme as well.